Update consignes-auteurs.md

spapastamkou opened the pull request:
*Mise à jour des consignes aux auteur(e)s - closes #2042 *

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [x] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

spapastamkou committed:
Update consignes-auteurs.md

spapastamkou commented:
J'ai commencé un pull request sous forme de "draft" pour qu'il soit plus aisé de réviser à plusieurs les consignes aux auteur(e)s selon #2042. Pas besoin de review dans l'immédiat.

InesSalome committed:
Update consignes-auteurs.md 01/05/21

Proposition de règles pour "dates et heures", "en-têtes" et "listes"

mariechristineb committed:
Update consignes-auteurs.md

mariechristineb commented on fr/consignes-auteurs.md:
```diff
@@ -35,12 +35,115 @@ Une fois que votre proposition est acceptée, un rédacteur ou une rédactrice v
+
+* Pour marquer les dates avant ou après Jésus-Christ, abréger sous la forme "av. J.-C." ou "apr. J.-C.".
+
+* Pour les heures, plusieurs options sont possibles :
```
J'aurais tendance à plutôt choisir un format pour les heures (ex.: 16 :10 ou 16h10), et un autre pour les durées (ex.: 10 heures 10 minutes)

spapastamkou replied:
peut-être ajouter "si possible" après "petites capitales" au cas où cela serait compliqué et qu'on aurait à l'implémenter en interne:)

mariechristineb commented on fr/consignes-auteurs.md:
```diff
@@ -35,12 +35,115 @@ Une fois que votre proposition est acceptée, un rédacteur ou une rédactrice v
+* Pour les heures, plusieurs options sont possibles :
+	* écrire les mentions "heures", minutes" et "secondes" en toutes lettres (10 heures 10 minutes)
+	* écrire toutes les mentions "heures" (h), minutes" (min) et "secondes" (s) en abrégé (10 h 10 min), avec un espace insécable entre l'abrévation et le chiffre
+	* écrire au format numérique avec un double point précédé d'un espace insécable (16 :10)
```
Il faudrait mentionner quelque part comment s'écrivent les espaces insécables

spapastamkou replied:
@mariechristineb: `&nbsp;`

InesSalome replied:
Merci à toutes les deux pour vos remarques, je viens de les prendre en compte ;)

InesSalome committed:
Update consignes-auteurs.md 09/05/21

Prise en compte des remarques de Marie-Christine et de Sofia dans la parte "Dates et heures".

spapastamkou committed:
mise a jour etape 1

spapastamkou committed:
nombres

spapastamkou committed:
correction coquilles dans nombres

InesSalome committed:
Ajout partie sur majuscules

À relire de mon côté pour harmoniser les informations et la mise en page.

InesSalome committed:
Mise à jour partie sur majuscules

InesSalome committed:
Améliorations section "Dates et heures"

InesSalome committed:
Correction coquilles section "Listes"

InesSalome committed:
Améliorations section "Majuscules"

InesSalome committed:
Accroche partie "Règles typographiques"

acrymble commented:
@spapastamkou I think this is done?

spapastamkou commented:
@acrymble no, my parts are missing among other things. I'll only be able to progress with this towards the end of this month.

mariechristineb committed:
ajout "écrire pour un public international"

mariechristineb committed:
Update consignes-auteurs.md

mariechristineb commented:
 @spapastamkou je viens d'ajouter une version révisée de la section "Écrire pour un public international". Je manquais un peu d'inspiration pour certains exemples, alors si quelqu'un de la @programminghistorian/french-team a des idées, elles sont bienvenues!

spapastamkou commented on fr/consignes-auteurs.md:
```diff
@@ -47,7 +47,20 @@ Cette partie aborde des questions générales de style pour vous aider à mieux
-#### Write for a Global Audience  
+### Écrire pour un public international
+
+Le lectorat du *Programming Historian en français* vient du monde entier et travaille au sein d'environnements culturels variés. De plus, les leçons des différentes publications du *Programming Historian* sont progressivement traduites depuis 2017, et ce, dans différentes combinaisons linguistiques. Nous avons d'ailleurs comme objectif d'en arriver à traduire toutes nos leçons dans la langue de c ...
```
"Nous avons d'ailleurs comme objectif d'en arriver à traduire toutes nos leçons dans la langue de chacune de nos publications, peu importe la langue de l'original." => @mariechristineb Tu n'as rien à voir avec cette partie, mais je crois bien qu'on pourra l'enlever: ce n'est pas du tout notre objectif de tout traduire ni d'aucune autre version langagière du PH:)

mariechristineb replied:
Je serais aussi d'accord pour enlever la dernière phrase, qui est effectivement trop ambitieuse. Peut-être qu'on pourrait seulement la remplacer par une phrase plus générale indicant le fait qu'il est possible, et souhaitable, que la leçon soit un jour traduite par une autre des publications.

spapastamkou replied:
A noter qu'on peut s'inspirer de la [récente discussion](https://github.com/programminghistorian/jekyll/issues/2143) qui a eu lieu sur les traductions.

spapastamkou committed:
update ponctuation

spapastamkou committed:
update références bibliographiques

spapastamkou commented:
Pour finir, il faudra un bon nettoyage du fichier pour enlever des parties obsolètes et uniformiser les derniers ajouts (je m'en charge et vous ferai signe si besoin de clarifications). Il faut aussi trancher dans les cas où les règles de ponctuation (notamment pour l'utilisation d'espaces insécables) ne sont pas les mêmes dans les pays francophones: je déposerai ici plus d'éléments d'information dans les jours qui viennent concernant ces problèmes.

spapastamkou commented:
@mariechristineb Je pense qu'il faudra qu'on revoit aussi la partie qui traite des "références à la race et aux origines ethniques" pour l'adapter mieux au contexte francophone. Le mot "race" est tabou, voire génère des polémiques, du coup mieux vaudra employer au plus "origines raciales et ethniques" (cela existe dans la législation). Pour rappel, dans la page "A propos" qui explique la politique de diversité du PH, nous avions utilisé à la place seulement "origines". On peut en reparler au moment du nettoyage du fichier.

mariechristineb commented:
> @mariechristineb Le mot "race" est tabou, voire génère des polémiques, du coup mieux vaudra employer au plus "origines raciales et ethniques" (cela existe dans la législation). Pour rappel, dans la page "A propos" qui explique la politique de diversité du PH, nous avions utilisé à la place seulement "origines". 

Merci @spapastamkou! J'étais mal à l'aise avec cette traduction un peu trop littérale de l'anglais, mais je n'arrivais pas 
à me souvenir de la terminologie correcte en français.

InesSalome committed:
Petit ajout section "Dates et Heures"

spapastamkou committed:
step 1 checked

spapastamkou committed:
adds language+style, open source + open access

spapastamkou committed:
Update consignes-auteurs.md

spapastamkou committed:
Update consignes-auteurs.md

spapastamkou committed:
ancres, caractères fr, markdown

spapastamkou committed:
Update consignes-auteurs.md

spapastamkou committed:
Update consignes-auteurs.md

spapastamkou committed:
update branch

spapastamkou committed:
corrige syntaxe

spapastamkou committed:
Update consignes-auteurs.md

spapastamkou committed:
fixes anchors and titles

spapastamkou committed:
coquilles typos, syntaxe

spapastamkou committed:
coquille et markdown

spapastamkou commented:
@InesSalome @matgille @datalogism @icolab Si vous consultez la prévisualisation, vous pouvez voir la version révisée des consignes aux auteurs. Pourriez-vous en faire une relecture et faire part de vos commentaires (si vous en avez) pour en tenir compte, le cas échéant? Il y a des ajouts spécifiques à la vf qu'il faut peut-être discuter (positionnement vis-à-vis des variantes du français, espacement avec certains signes de ponctuation...). Merci!

datalogism commented:
Hello @spapastamkou, j'ai repéré quelques coquilles, si tu le souhaites je peux aller directement les corriger :
* La portée et la longueur du tutoriel doivent être **appropriés** ->  approprié**e**s
* Ce guide de style présente notre approche et notre charte éditoriale à l'attention des auteur(e)s. **En en** respectant **les** principes, vous nous aidez à conserver la cohérence des contenus du Programming Historian en français. 
  *  double **en**
  *  les -> ces
* de bien lire cette partie lorsque vous vous **projettez** dans l'écriture d'une leçon
* À tout moment, en cas de doute, vous pouvez vous adresser au membre de notre équipe qui assure le suivi éditorial de votre leçon pour obtenir plus de clarifications, si **nécesaire** -> nécessaire
* Aussi,fournissez des conseils généraux -> espace
*  Vous devez avoir le droit de redistribuer une copie de ces données et **les** mettre à disposition dans un format ouvert. -> "de les"
* « av. J.-C. »ou « apr. J.-C. ». -> espace
* /!\ plusieurs fois :  **une** espace insécable
* **derner** item
* OCDE pour Organisation de coopération et de développement **économiques**
*  **intepellation**
* en Belgique **lil** n'y a pas d'espacement
* /!\ espace après ":"
* et **précéde** 
* les règles ci-**des« us** 
* au **permier** mot
*  villes prennent une **majsucule**, 
*  mots **relevanat** de la religion
* personne précise, auquel cas **pas** il faut éviter 
* Si votre tutoriel a vocation **d'**être un tutoriel d'analyse -> à être
* nous **ne faisons** pas recours -> n'avons pas recours
* Veillez à **utiiser** un format adapté à
* et **le** communiquer -> "de le "
* lecture attentive **en en** testant les manipulations proposées -> double en
* pourvu que les règles de bonne conduite, **explicités** dans le ticket -> explicitées
* Parfois -> virgule après
* votre leçon publiée? -> espace
* Nous oeuvrons  -> œuvrons

spapastamkou commented:
merci beaucoup pour ta vigilance, @datalogism, cela fait un bon paquet! Si tu veux, tu peux intervenir directement et, si tu n'as pas la possibilité, je les passerai avant de fusionner le PR. Encore merci!

spapastamkou commented:
Concernant vos commentaires dans #2312 @datalogism et @InesSalome, peut-être nous pouvons réfléchir tous ensemble et faire une petite mise à jour le cas échéant des consignes aux traducteurs/traductrices (dans un 2e temps)?

matgille commented:
Bonjour, je vais essayer de jeter un coup d'oeil ce week end, je n'aurai pas le temps avant !



Matthias

InesSalome commented:
Comme Matthias je ne pourrai pas relire avant ce week-end, la semaine prochaine au plus tard !

anisa-hawes commented:
Dear @spapastamkou, It's great to see how much progress has been made! I notice that all the status checks have passed. Are you ready to merge this? 

You mention a 'second step' above relating to the Translator's Guidelines. If I understand correctly, this could be a future piece of work for the FR team. 

Let me know if I can help in any way.

datalogism committed:
misprint check and correct

see my message in the #2114  thread

datalogism commented:
> 
> 
> merci beaucoup pour ta vigilance, @datalogism, cela fait un bon paquet! Si tu veux, tu peux intervenir directement et, si tu n'as pas la possibilité, je les passerai avant de fusionner le PR. Encore merci!

;) Done

InesSalome commented:
Voici mes remarques (parfois tatillonnes ;) ) : 

- À la place de "qui portent sur un problème de recherche ou un processus particulier" dans le 1er paragraphe sous la section "Étape 1" (l. 20 du fichier markdown), j'aurais tendance à remplacer "un problème" par "une problématique de recherche".
- Dans la section "Écrire pour un public international", je remplacerais "tout mot qu'un(e) citoyen(ne) moyen(ne) peut ne pas connaître ou comprendre" par "tout mot qu'un(e) citoyen(ne) non-initié(e) peut ne pas connaître ou comprendre" (l. 59).
- Dans la section "Écrire pour un public international", "sans oublier qu'il peut exister de différentes perceptions et dénominations d'un pays à l'autre" (l. 60), il y a un "de" en trop. 
- Dans la section "Ponctuation", "Préférer d'écrire les sigles sans points abréviatifs et en majuscules non accentués" (l. 144), je commencerais la phrase plutôt apr "Privilégier l'écriture des sigles...".
- Dans la section sur les majuscules, coquille avec un guillemet en trop "(exemple : le ministère des Affaires étrangères).« " (l. 207). 

Si tu es d'accord avec mes remarques @spapastamkou, je peux aller faire les modifications directement :)

spapastamkou commented:
@datalogism Merci infiniment à toi!
@InesSalome Merci beaucoup, tu peux y aller sans problème. 
Dans l'avenir, n'hésitez pas, c'est notre travail collectif:)

spapastamkou committed:
Merge branch 'gh-pages' into consignes-auteurs-revision

spapastamkou committed:
Create modele-tuto.md

ajout fichier modèle de leçon

spapastamkou committed:
lien fichier modele de tuto

spapastamkou commented:
@programminghistorian/spanish-team After updating the branch, there appears to be a link problem in a Spanish lesson: /es/lecciones/introduccion-datos-abiertos-enlazados.html 
```http://dbpedia.org/snorql/ failed: 404 No error ```
The page returns: ```Error HTTP/1.1 404 File not found
The requested URL is not an ordinary resource    URI  = '/snorql/' ```
There is this archived version on Internet Archive: https://web.archive.org/web/20210506201504/https://dbpedia.org/snorql/ 
I will be happy to change the link in the lesson but I need someone to tell me if the lesson is still functionnal after the change. 
This concerns also the @programminghistorian/english-team and the following lesson [Introduction to the Principles of Linked Open Data](https://programminghistorian.org/en/lessons/intro-to-linked-data) that does not cause the builds to fail yet but undoubtedly will. 
Thanks!

anisa-hawes commented:
Thank you, @spapastamkou. How about this as an alternative? https://www.dbpedia.org/resources/sparql/

acrymble commented:
@spapastamkou I think you can make this a Lesson Maintenance issue which @anisa-hawes can help resolve in her role. A broken link is a lesson maintenance problem, after all.

InesSalome committed:
amendements après relecture

InesSalome commented:
> @datalogism Merci infiniment à toi! @InesSalome Merci beaucoup, tu peux y aller sans problème. Dans l'avenir, n'hésitez pas, c'est notre travail collectif:)

C'est fait ! :)

spapastamkou commented:
@InesSalome grand merci!

spapastamkou commented:
Le contenu de cette page est ok; une fois le problème du build résolu, il faudra juste qu'un des membres de l'équipe fr approuve puis il sera tout à fait possible de fusionner. 
The content of this page is stable, so once technically possible it only needs a review and then can be merged (I think this answers [your previous question](https://github.com/programminghistorian/jekyll/pull/2114#issuecomment-953062725) @anisa-hawes).

matgille commented:
Bonjour à tout le monde, 

Voici quelques remarques suite à ma relecture tardive (désolé): 
- En typographie il me semble que l'espace est un nom féminin:  `L’utilisation d’un espace insécable fine avant le point d’exclamation est recommandée en France` > `L’utilisation d’**une** espace insécable fine avant le point d’exclamation est recommandée en France`; il y a d'autres occurrences à modifier. 
- Je remonterais le style bibliographique choisi par PH en premier item de la liste de la section `Références bibliographiques`, pour mieux le mettre en valeur. Éventuellement, je le sortirais de la liste pour le mettre bien en avant. 
- Je suppose que  `rude` est traduction littérale de l'anglais: je propose `incorrect(e)`.

Si vous acceptez ces suggestions, je m'occuperai de modifier le fichier !


Bonne semaine, 


Matthias

spapastamkou commented:
Merci, Matthias! Si tu peux intégrer directement, c’est parfait!

Envoyé de mon iPhone

> Le 3 nov. 2021 à 11:19, Matthias Gille Levenson ***@***.***> a écrit :
> 
> ﻿
> Bonjour à tout le monde,
> 
> Voici quelques remarques suites à ma relecture tardive:
> 
> En typographie il me semble que l'espace est un nom féminin: L’utilisation d’un espace insécable fine avant le point d’exclamation est recommandée en France > L’utilisation d’**une** espace insécable fine avant le point d’exclamation est recommandée en France; il y a d'autres occurrences à modifier.
> Je remonterais le style bibliographique choisi par PH en premier item de la liste de la section Références bibliographiques, pour mieux le mettre en valeur. Éventuellement, je le sortirais de la liste pour le mettre bien en avant.
> Je suppose que rude est traduction littérale de l'anglais: je propose incorrect(e).
> Si vous acceptez ces suggestions, je m'occuperai de modifier le fichier !
> 
> Bonne semaine,
> 
> Matthias
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android.

matgille committed:
Update consignes-auteurs.md

Amendements mineurs après relecture

matgille commented:
@spapastamkou Voilà qui est fait !

Matthias

anisa-hawes commented:
Very many thanks all @programminghistorian/french-team! I am working on solving the broken link issue, and will merge this for you ASAP.

anisa-hawes commented:
Hello @acrymble. Would you be happy to review this PR? As the French team have worked together to prepare the work, we can be confident the language is correct! 

I've checked the Preview on Netlify and it looks good from a formatting point of view.

spapastamkou commented:
@anisa-hawes thanks for dealing with this PR. However, the new guidelines as prepared in this branch are not currently live on the site. From what I can understand the PR has not been reviewed and merged here, but there is merge and a revert in #2355? Please let me know if I get it right (or not!), and in any case I reopened the issue #2042 that we can close once the guidelines are live. Thank you very much!
